Armchair.xcodeproj/xcshareddata/xcschemes/Armchair.xcscheme

```diff
@@ -28,16 +28,6 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
-         <TestableReference
-            skipped = "NO">
-            <BuildableReference
```

Comment:
Can you explain what is happening here? Was this automatic or did you manually remove these?

Reply:
They had a phantom test target left in their scheme. We don't really need this change one way or another, however, I plan to open a PR back to upstream Urban when I clean it up more. For now, these changes accomplish what we need and this is just extra

Reply:
Gotcha! Thanks for the explanation!